dbdimp.c

```diff
@@ -5422,7 +5422,6 @@ static int to_numeric(char *str, SV *sth, imp_sth_t *imp_sth, CS_DATAFMT *datafm
   CS_DATAFMT srcfmt;
   CS_INT reslen;
   char *p;
-  CS_LOCALE *locale = LOCALE(imp_dbh);
```

Comment:
Ah - now I get it. The problem really is that there is a global variable called "locale" as well, so the macro here is taking the wrong variable.
Ugh - really ugly....

Thanks!